docs/scheduler-http-api.md

```diff
@@ -128,6 +128,26 @@ TEARDOWN Response:
 HTTP/1.1 202 Accepted
 ```
 
+### SUPPRESS
```

Comment:
Can you move this to #500 below #REQUEST section to preserve the call enum order?